test-suite/src/test/scala/org/scalajs/testsuite/javalib/CollectionTest.scala

```diff
@@ -165,7 +165,7 @@ trait CollectionTest extends JasmineTest with ExpectExceptions {
       coll.add("one")
       expect(coll.contains("one")).toBeTruthy
       expect(coll.contains("two")).toBeFalsy
-      expect(coll.contains(null)).toBeFalsy
+      //expect(coll.contains(null)).toBeFalsy
```

Comment:
It would also be a good idea to add the `def allowsNullElement: Boolean` to `CollectionFactory` and test that we can add a `null` element. If so and if `allowsNullElementQuery == true` we can also test that `.contains(null) == ture` in this corner case.